src/components/EcoCounter/EcoCounterContent/EcoCounterContent.js

```diff
@@ -430,11 +430,17 @@ const EcoCounterContent = ({
     } else setUserTypesList(bicycleOnly);
   }, [stationId]);
 
+  const renderStationName = (input) => {
+    if (input === 'Teatteri ranta') {
```

Comment:
Pretty specific fix. Are there other similar cases? Should really be fixed at the data source, this is data source related issue

Reply:
It originates from .csv -file. It would be better to fix it at the data source. This is the only case.